Source/WTF/wtf/Compiler.h

```diff
@@ -468,12 +382,8 @@
 
 /* UNUSED_VARIADIC_PARAMS */
 
-#if !defined(UNUSED_VARIADIC_PARAMS) && (COMPILER(GCC) || COMPILER(CLANG))
-#define UNUSED_VARIADIC_PARAMS __attribute__((unused))
-#endif
-
 #if !defined(UNUSED_VARIADIC_PARAMS)
-#define UNUSED_VARIADIC_PARAMS
+#define UNUSED_VARIADIC_PARAMS __attribute__((unused))
```

Comment:
Looks like the last one to remove from the previous patch is the
```
#if !defined(WARN_UNUSED_RETURN)
#define WARN_UNUSED_RETURN
#endif
```
right below here.